src/main/java/com/yelp/nrtsearch/server/handler/SearchHandler.java

```diff
@@ -328,7 +328,8 @@ public SearchResponse handle(IndexState indexState, SearchRequest searchRequest)
                         InnerHitFetchTask::getDiagnostic)));
       }
       searchContext.getResponseBuilder().setDiagnostics(diagnostics);
```

Comment:
I think the diagnostics object here contains all of the required metrics, including the rescorer metrics. Can you please confirm once?